src/controllers/user.controller.js

```diff
@@ -169,6 +169,114 @@ class UserController {
       return responseGenerator.sendError(res, 500, error.message);
     }
   }
-}
 
+  /**
```

Comment:
Great work so far @codexempire  but I do not think you made any changes in the `usersController`. Why does it appear such? Please make amends

Reply:
Hello @codexempire , Please click [here](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git).

I also posted a link to an article that might help with rebase on one of my previous comments.